kubernetes-examples/multitenant/postinstall.sh

```diff
@@ -7,7 +7,7 @@
 
 set -eo pipefail
 
-STRIMZI_KAFKA=quay.io/strimzi/kafka:0.35.0-kafka-3.4.0
+STRIMZI_KAFKA=quay.io/strimzi/kafka:0.38.0-kafka-3.6.0
```

Comment:
in quay there is also are also tags like `latest-kafka-3.6.0` but figured we should pick a more stable tag.